components/script/dom/webglrenderingcontext.rs

```diff
@@ -4372,7 +4384,10 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
         source: TexImageSource,
     ) -> ErrorResult {
         if !self.extension_manager.is_tex_type_enabled(data_type) {
-            return Ok(self.webgl_error(InvalidEnum));
+            return {
+                self.webgl_error(InvalidEnum);
+                Ok(())
+            };
```

Comment:
It feels like this should be spit up into two statements instead of putting multiple statements in a block after `return`. Please make this change here and other place the same pattern is repeated.